notebooks/Liquidators_data.ipynb

```diff
@@ -0,0 +1,1325 @@
AAAAAKLQcbsgytG3bVm3bts3NXAAAAACgSHGoIUtPT9fChQv1+eef6/jx4zIMQxUqVNB9992niIgIGYaRV3kCAAAAQKFj9zVkpmmqZ8+eGjJkiP744w/Vrl1bNWvW1IkTJzRw4EDdfffdeZknAAAAABQ6ds+QLVy4UD/88IM2btyoNm3a2Gz77rvv1Lt3by1evFj9+/fP9SQBAAAAoDCye4bso48+0nPPPZelGZOuX082btw4LV26NFeTAwAAAIDCzO6GbO/evercuXOO27t06aJffvklV5ICAAAAgKLA7obs0qVLCgkJyXF7SEiIYmNjcyUpAAAAACgK7G7I0tLS5OaW8yVnrq6uSk1NzZWkAAAAAKAosHtRD9M0NXDgQHl6ema7PSkpKdeSAgAAAICiwO6GbMCAAbccwwqLAAAAAGA/uxuyBQsW5GUeAAAAAFDk2H0NGQAAAAAgd9GQAQAAAICT0JABAAAAgJPQkAEAAACAk9CQAQAAAICT2LXK4qpVq+zeYc+ePf92MgAAAABQlNjVkPXu3dvmtmEYMk3T5naGtLS03MkMAAAAAAo5u05ZTE9Pt/5bv3696tWrp6+//lpxcXGKj4/X2rVr1aBBA61bty6v8wUAAACAQsPuL4bOMGrUKL377rtq2bKlNdapUyf5+Pho2LBhOnDgQK4mCAAAAACFlcOLehw5ckQWiyVL3GKx6Pjx47mREwAAAAAUCQ43ZI0bN9aoUaMUHR1tjcXExGjMmDG68847czU5AAAAACjMHG7I5s+fr3Pnzql8+fKqXLmyKleurHLlyik6Olrz5s3LixwBAAAAoFBy+BqyypUra+/evdqwYYMOHjwo0zRVo0YNtW/f3ma1RQAAAADAzTnckEnXl7nv2LGj7rrrLnl6etKIAQAAAMDf4PApi+np6XrxxRdVunRpFStWTMeOHZMkjR8/nlMWAQAAAMABDjdkL730khYuXKjp06fLw8PDGq9du7Y++OADh/b1ww8/qEePHgoLC5NhGPriiy9stg8cOFCGYdj8a9q0qc2YpKQkPf744woMDJSvr6969uyp06dP24yJjY1VRESELBaLLBaLIiIiFBcXZzPm5MmT6tGjh3x9fRUYGKiRI0cqOTnZoXoAAAAAwBEON2SLFy/We++9pwcffFCurq7WeJ06dXTw4EGH9pWYmKi6detqzpw5OY7p3LmzoqOjrf/Wrl1rs33UqFFauXKlli9fri1btujKlSvq3r270tLSrGP69eunqKgorVu3TuvWrVNUVJQiIiKs29PS0tStWzclJiZqy5YtWr58uVasWKExY8Y4VA8AAAAAOMLha8j++OMPVa5cOUs8PT1dKSkpDu2rS5cu6tKly03HeHp6KjQ0NNtt8fHxmjdvnpYsWaL27dtLkj788EOVLVtW3377rTp16qQDBw5o3bp12r59u5o0aSJJev/999WsWTMdOnRI4eHhWr9+vX777TedOnVKYWFhkqSZM2dq4MCBevnll1W8eHGH6gIAAAAAezjckNWsWVM//vijypcvbxP/9NNPVb9+/VxLLMOmTZsUHBwsf39/tWrVSi+//LKCg4MlSXv27FFKSoo6duxoHR8WFqZatWpp69at6tSpk7Zt2yaLxWJtxiSpadOmslgs2rp1q8LDw7Vt2zbVqlXL2oxJUqdOnZSUlKQ9e/aoTZs22eaWlJSkpKQk6+2EhARJUmpqqlJTUyVJLi4ucnFxUXp6utLT061jM+JpaWkyTfOWcVdXVxmGYd1v5rgkmxnBm8Xd3NxkmqZN3DAMubq6Zskxpzg1URM15b+aZJqSYchIt83RNK6fCGGY6fbFXVwl07SNG8b18TnG02VkysU0DOkmccNMv55v5lwMI+c4NVFTbtQkFenXCGqiJmq6/TXduD0nDjdkEyZMUEREhP744w+lp6fr888/16FDh7R48WJ99dVXju7uprp06aJ///vfKl++vI4dO6bx48erbdu22rNnjzw9PRUTEyMPDw8FBATY3C8kJEQxMTGSrn9pdUYDl1lwcLDNmJCQEJvtAQEB8vDwsI7JztSpUzVp0qQs8cjISPn6+kqSgoKCVKlSJR07dkznz5+3jilTpozKlCmj33//XfHx8dZ4xYoVFRwcrH379unq1avWeLVq1eTv76/IyEibJ2KdOnXk4eGh3bt32+TQqFEjJScna+/evdaYq6urGjdurPj4eJvTS729vVW3bl1duHBBR48etcYtFouqV6+uM2fO2FyXR03URE35ryYvI1TXPIop7NJhGZn+yMSUqKQ0FzeVvnDIpqY/AsPlmp6q0EtHrDHTxUV/BFaTV0qiAuNOWuOpbp6KKVFJvtfiFHA52hq/5uGrC/7lVfzPiyqe+Ffuid7+ivULU8CVGPlejbPGE3yDlOAbpJLxp+SVnGiNx/qVUqJ3gEJij8kt9a8PuS74l6Mmasq1mqTgIv0aQU3URE23v6bExL9eQ2/GMG0+YrXPN998oylTpmjPnj1KT09XgwYN9MILL9jMVDnKMAytXLlSvXv3znFMdHS0ypcvr+XLl+uee+7RsmXLNGjQIJtZKknq0KGDKlWqpHfffVdTpkzRokWLdOiQ7R+EKlWqaPDgwRo3bpyGDRumEydO6JtvvrEZ4+HhocWLF6tv377Z5pPdDFnZsmV18eJF62mOfLpATdRETbejphl7YwvGLEVhnHmhpgJR07gGwUX6NYKaqImabn9NCQkJKlmypOLj4296CdTf+h6yTp06qVOnTn/nrv9IqVKlVL58eR0+fFiSFBoaquTkZMXGxtrMkp07d07Nmze3jjl79myWfZ0/f946KxYaGqodO3bYbI+NjVVKSkqWmbPMPD095enpmSXu5uYmNzfbH23GL/RGmRdGsSd+437/TtwwjGzjOeXoaJyaqCmnODXlYU3/f0qW6ZL9eNNwIG4YDsZdZGb3dZQ5xK+/gXcgTk3UlEs1FenXiJvk6GicmqhJoqaccswcz2l7lnz
```

Comment:
This chart contains all information as some of the charts above + something extra (distribution between tokens). Pls consolidate the charts, so that only the one with richer information remains.